consensus/hotstuff/signature/weighted_signature_aggregator.go

```diff
@@ -19,28 +18,40 @@ type signerInfo struct {
 
 // WeightedSignatureAggregator implements consensus/hotstuff.WeightedSignatureAggregator
 type WeightedSignatureAggregator struct {
-	aggregator   *signature.SignatureAggregatorSameMessage // low level crypto aggregator, agnostic of weights and flow IDs
-	signers      []flow.Identity                           //nolint:unused
-	idToSigner   map[flow.Identifier]signerInfo
-	totalWeight  uint64                       // weight collected
-	lock         sync.RWMutex                 // lock for atomic updates
-	collectedIDs map[flow.Identifier]struct{} // map of collected IDs
+	aggregator   *signature.SignatureAggregatorSameMessage // low level crypto BLS aggregator, agnostic of weights and flow IDs
+	ids          []flow.Identity                           //nolint:unused (only gets updated by constructor)
+	idToInfo     map[flow.Identifier]signerInfo            // (only gets updated by constructor)
+	totalWeight  uint64                                    // weight collected (gets updated)
+	collectedIDs map[flow.Identifier]struct{}              // map of collected IDs (gets updated)
+	lock         sync.RWMutex                              // lock for atomic updates to totalWeight and collectedIDs
 }
 
-var _ hotstuff.WeightedSignatureAggregator = &WeightedSignatureAggregator{}
```

Comment:
I would prefer to have it, it's a useful thing to have

Reply:
I'll put it back, but what is it useful for?